src/components/link-preview/link-preview.stories.mdx

```diff
@@ -0,0 +1,71 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+
+import LinkPreview from '.';
+
+<Meta
+  title="Design System/Link Preview"
+  parameters={{ info: { disable: true }}}
+/>
+
+# Message
```

Comment:
Should this be `Message`? The component is called `Link Preview`.

Reply:
Copy and paste 🤦